tests/plugin_utils_spec.lua

```diff
@@ -1,39 +1,42 @@
-local a = require('plenary.async_lib.tests')
```

Comment:
Many undesirable changes are brought by an auto format feature.
It can be remove if needed.